tests/core/test_cli_entry.py

```diff
@@ -49,7 +49,7 @@ def is_valid():
         # Make sure parsing and validation is done
         mock_parsing_handler.parse_and_validate.assert_called_with(directory)
         # api must be initialized
-        mock_api_handler.initialize_api_from_config.assert_called()
+        mock_api_handler.initialize_api_from_config.assert_called_with()
```

Comment:
`assert_called` was added in Python 3.6, using `assert_called_with` with no arguments is functionally the same in our use cases and allows compatibility with 3.5.